airflow/providers/slack/operators/slack.py

```diff
@@ -30,7 +30,7 @@
 
 class SlackAPIOperator(BaseOperator):
     """
-    Base Slack Operator
+    Base Slack Operator.
```

Comment:
Needs a newline.

Reply:
Somehow docs building was failing because of that. I reverted then